### PR TITLE
feat: added avalanche assets starting with USDC & EUROC

### PR DESCRIPTION
**Pull request by schystz**

### CODE REVIEW

File: blockchains/avalanche/assets/0xdB25f211AB05b1c97D595516F45794528a807ad8/info.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (0xAplki) [inline blockchains/avalanche/assets/0xdB25f211AB05b1c97D595516F45794528a807ad8/info.json]: note sure why this `https://snowtrace.io/address/0xC891EB4cbdEFf6e073e859e987815Ed1505c2ACD` does not match `blockchains/avalanche/assets/0xdB25f211AB05b1c97D595516F45794528a807ad8` ?

**Review by 0xAplki — COMMENTED** [review]
one minor clarification

File: blockchains/avalanche/assets/0xdB25f211AB05b1c97D595516F45794528a807ad8/info.json (hunk 1 above)
Comment (schystz) [reply blockchains/avalanche/assets/0xdB25f211AB05b1c97D595516F45794528a807ad8/info.json]: Ah thanks for the catch, this is EURS address on mainnet where I copied the folder from. I forgot to rename the folder itself, my bad!

Comment (schystz) [reply blockchains/avalanche/assets/0xdB25f211AB05b1c97D595516F45794528a807ad8/info.json]: Fixed on https://github.com/xave-finance/assets/pull/10/commits/b3c7bced1710ae620def4d98fcbdca9ef232898b @0xAplki !

File: blockchains/avalanche/assets/0xC891EB4cbdEFf6e073e859e987815Ed1505c2ACD/info.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Euro Coin",
+  "website": "https://www.circle.com/en/euro-coin",
+  "description": "Euro Coin, or EUROC, is a euro-backed stablecoin that’s accessible globally on Ethereum and Avalanche.",
+  "socials": [],
+  "explorer": "https://snowtrace.io/address/0xC891EB4cbdEFf6e073e859e987815Ed1505c2ACD",
```
Comment (tracyarciaga) [inline blockchains/avalanche/assets/0xC891EB4cbdEFf6e073e859e987815Ed1505c2ACD/info.json]: @schystz the one above is /token, this one is /address. I guess it doesn't matter?

**Review by tracyarciaga — APPROVED** [review]

File: blockchains/avalanche/assets/0xC891EB4cbdEFf6e073e859e987815Ed1505c2ACD/info.json (hunk 2 above)
Comment (schystz) [reply blockchains/avalanche/assets/0xC891EB4cbdEFf6e073e859e987815Ed1505c2ACD/info.json]: Yeah...but fixed it anyway on https://github.com/xave-finance/assets/pull/10/commits/8a260d2f6d3482ec43ee98dfef0c29b9b3f1310a so its consistent! Thanks for the catch!

**Review by 0xAplki — APPROVED** [review]

**Pull request merged**